ssh/tailssh/incubator.go

```diff
@@ -249,14 +230,15 @@ func beIncubator(args []string) error {
 		}
 	}
 
-	euid := os.Geteuid()
-	runningAsRoot := euid == 0
-	if runningAsRoot && ia.loginCmdPath != "" {
-		// Check if we can exec into the login command instead of trying to
-		// incubate ourselves.
-		if la := ia.loginArgs(); la != nil {
-			return unix.Exec(ia.loginCmdPath, la, os.Environ())
-		}
+	if ia.isSFTP {
+		return dropPrivilegesAndHandleSFTP(logf, ia)
```

Comment:
doesn't this also need to go through the login command to make sure the home dir is created?

Reply:
You are correct.

Reply:
This adds another layer of complexity, as we'll probably need to respawn ourselves again, this time calling ourselves with `su` and running just as an ftp server...  I'm tempted to do that in a separate PR.

Reply:
that seems fine, leave a TODO at least?

Reply:
I tried playing with this, and it won't work because tailscaled requires root, so we can't `su` as the actual user to run tailscaled.

I suppose we could launch a throwaway shell just to trigger pam_mkhomedir, but that doesn't work for other pam stuff like pam_sssd.

So yeah, for now I'll just leave a comment.